woocommerce/woocommerce
# Include Kosovo (XK) in countries, states and continents

### All Submissions:


### Changes proposed in this Pull Request:

Added Kosovo 'XK' under countries, continents (part of Europe) and states (a country with no States). Keeping it listed as a subregion of Serbia as well per [this comment](https://github.com/woocommerce/woocommerce/issues/35819#issuecomment-1404445203
)

Closes #35819 .


### To test:

1)

- test without this branch
- go to shipping zones
- add shipping zone
- attempt to add Kosovo as a Zone Region
- You currently only see Kosovo, Serbia

Then

- test with this branch
- go to shipping zones
- attempt to add Kosovo as a Zone Region
- You see Kosovo, Serbia and also Kosovo as a standalone country

2)

- test without this branch
- go to WooCommerce > Settings > General
- attempt to add kosovo as the store Country/State
- You currently only see Serbia, Kosovo

Then

- test with this branch
go to WooCommerce > Settings > General
- attempt to add kosovo as the store Country/State
- You see Serbia, Kosovo and also Kosovo as a standalone country


### Other information:


### FOR PR REVIEWER ONLY:
